app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -57,9 +62,30 @@ public void goToHomePage() {
         startActivity(intent);
     }
 
+    @Override
+    public void saveLogin(String email, String password) {
```

Comment:
This should not be done in the fragment. Remember that all the logic is done by the presenter. In fact, the Fragment should not have access to sharedPreferences